.rubocop_todo.yml

```diff
@@ -17,6 +17,12 @@ Lint/NonDeterministicRequireOrder:
   Exclude:
     - 'spec/rails_helper.rb'
 
+# Offense count: 1
+# Cop supports --auto-correct.
+Lint/RedundantCopDisableDirective:
```

Comment:
This seems odd, can we just remove the offense?

Reply:
will do